recipes/doxygen/all/test_package/conanfile.py

```diff
@@ -6,13 +6,13 @@ class TestPackageConan(ConanFile):
     generators = "cmake"
 
     def build(self):
-        if not tools.cross_building(self.settings, skip_x64_x86=True):
+        if not tools.cross_building(self, skip_x64_x86=True):
             cmake = CMake(self)
             cmake.configure()
             cmake.build()
```

Comment:
```suggestion
            with tools.RunEnvironment(self):
                cmake = CMake(self)
                cmake.configure()
                cmake.build()
```

Reply:
Looks like this change was already in `master` 🚀